tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -110,17 +110,17 @@ public synchronized RegionManager getRegionManager() {
   public synchronized ManagedChannel getChannel(String addressStr) {
     ManagedChannel channel = connPool.get(addressStr);
     if (channel == null) {
-      HostAndPort address;
+      URI address;
       try {
-        address = HostAndPort.fromString(addressStr);
+        address = URI.create("http://" + addressStr);
```

Comment:
Maybe you can move it to the field variable?

Reply:
fixed in #589